frontend/public/json-example/communitys.json

```diff
@@ -0,0 +1,7 @@
+{
+  "community": [
+    {"id": 1, "name": "Community 1", "location": "Ort 1", "description":"description 1 description 1 description 1 description 1 description 1 description 1 description 1 ", "url": "http://localhost:3000/vue/"},
```

Comment:
I asked myself if we want to put this in an apollo call so we have apollo responsible for this.
What do you think?

Reply:
that should be decided by @ulfgebhardt

Reply:
We can put this into an apollo call, but since we dont have any other data to feed to the client - just static stuff i thought it would simplify things if we just keep it client till we have the infrastructure to serve an actual list from a p2p-net (or blockchain for that matter).
Tho you are right. In the original Issue of claus-peter he defines that we should implement a server call + a database table for this data.
I guess you decide or we do an refinement